src/Toolchain/DefaultToolchain.ts

```diff
@@ -56,9 +56,10 @@ class DefaultToolchain {
     this._toolchainEnv = toolchainEnv;
     this._toolchain = toolchain;
     const name = `${toolchain.info.name}-${toolchain.info.version ?.str()}`;
-    vscode.window.showInformationMessage(`Set ${name} as a default toolchain.`, 'OK', `Instruction`)
+    vscode.window
+        .showInformationMessage(`${name} was set as a default toolchain.`, 'OK', `Instructions`)
```

Comment:
How about this?
I think most vscode buttons on information messages has "Do something" format. ex. Open something, View logs, Reload window, etc,,,

```suggestion
        .showInformationMessage(`${name} was set as a default toolchain.`, 'OK', `See Instructions`)
```